fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1754,6 +1754,9 @@ public class Config extends ConfigBase {
     @ConfField
     public static int lake_compaction_max_tasks = -1;
 
+    @ConfField(mutable = true)
+    public static boolean lake_verbose_compaction_log = false;
```

Comment:
not necessary to add this config, you can set `sys_log_verbose_modules = com.starrocks.lake.compaction` to print lake compaction debug logs.